lib/tramway/base_form.rb

```diff
@@ -65,7 +65,7 @@ def respond_to_missing?(method_name, include_private = false)
 
     def __submit(params)
       self.class.properties.each do |attribute|
-        public_send("#{attribute}=", params[attribute])
+        public_send("#{attribute}=", params[attribute]) if params.keys.include? attribute.to_s
```

Comment:
in this case, attributes won't be updated to empty values. The main purpose of this update is to prevent updating attributes not included in params. But if an attribute is included and it's nil (or empty) then it must be updated.